v0.5/recommendation/README.md

```diff
@@ -0,0 +1,262 @@
+# MLPerf Inference Benchmarks for Recommendation Task
+
+This is the reference implementation for MLPerf Inference benchmarks.
+
+### Supported Models
+
+| model | framework | accuracy | AUC | dataset | trained  | precision | notes |
+| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
+| dlrm | PyTorch | 78.9% | N/A | [Criteo Kaggle DAC](https://labs.criteo.com/2014/02/kaggle-display-advertising-challenge-dataset/)       | N/A                                                                     | fp32 |                          |
+| dlrm | PyTorch | 81.07% | N/A | [Criteo Terabyte](https://labs.criteo.com/2013/12/download-terabyte-click-logs/) (0.875) | [weights](https://dlrm.s3-us-west-1.amazonaws.com/models/tb0875_10M.pt) | fp32 | --max-ind-range=10000000 --data-sub-sample-rate=0.875 |
+| dlrm | PyTorch | N/A | 80.25% | [Criteo Terabyte](https://labs.criteo.com/2013/12/download-terabyte-click-logs/)         | [weights](https://dlrm.s3-us-west-1.amazonaws.com/models/tb00_40M.pt)   | fp32 | --max-ind-range=40000000 |
+
+### Disclaimer
+This benchmark app is a reference implementation that is not meant to be the fastest implementation possible.
+It is written in python which might make it less suitable for large number of CPU's.
+
+The reference implementation includes all required pre-processing of datasets.
+It also includes a ```--accuracy``` option to validate accuracy and AUC as required by mlperf.
```

Comment:
Please explain what AUC is.

Reply:
AUC is a standard metric. I have added a reference that explains it.

Reply:
Very helpful, thanks. You are the expert. We are learning... :)